wordle-web/components/navigation-dialog.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div>
+    <v-btn icon @click="toggleDialog">
+      <v-icon> mdi-hamburger</v-icon>
+    </v-btn>
+
```

Comment:
I would recommend a `v-menu` or something similar below rather than a dialog, as it's a more common paradigm.

See docs here: https://vuetifyjs.com/en/components/menus/

Something like this for example:
```vue
<template>
  <v-menu offset-y>
    <template #activator="{ on, attrs }">
      <v-btn icon v-bind="attrs" v-on="on">
        <v-icon> mdi-hamburger</v-icon>
      </v-btn>
    </template>

    <v-list dark>
      <v-list-item-group>
        <v-list-item nuxt to="/">
          <v-list-item-title> Home </v-list-item-title>
        </v-list-item>
        <v-list-item nuxt to="/game">
          <v-list-item-title> Game </v-list-item-title>
        </v-list-item>
        <v-list-item nuxt to="/about">
          <v-list-item-title> About </v-list-item-title>
        </v-list-item>
      </v-list-item-group>
    </v-list>
  </v-menu>
</template>
```